### PR TITLE
add extension info to struct field

### PR DESCRIPTION
**Pull request by lutaoact**

### CODE REVIEW

**Comment by Nerzal** [comment]
Could you add a test method for this feature?

**Comment by lutaoact** [comment]
@Nerzal I have editted the test files to cover my code.

File: README.md [hunk 1]
```diff
@@ -572,5 +572,30 @@ type Account struct {
...
+        "id": {
+            "type": "string",
+            "x-nullable": true,
+            "x-abc": "def"
+        }
+    }
+}
+```
+
+Maybe some other tools will use extension info. For example, [go-swagger/go-swagger](https://github.com/go-swagger/go-swagger) use `x-nullable` to render struct field as pointer when generate client. [reference](https://goswagger.io/use/models/schemas.html)
```
Comment (easonlin404) [inline README.md]: @lutaoact can you delete this paragraph? 

> Maybe some other tools will use extension info. For example, [go-swagger/go-swagger]

Comment (lutaoact) [reply README.md]: @easonlin404 ok, done.

**Pull request merged**